libs/coin-modules/coin-tron/src/logic/estimateFees.test.ts

```diff
@@ -0,0 +1,31 @@
+import { estimateFees } from "./estimateFees";
+import { ACTIVATION_FEES, ACTIVATION_FEES_TRC_20 } from "./constants";
+import { TransactionIntent } from "@ledgerhq/coin-framework/api/index";
+
+describe("estimateFees", () => {
```

Comment:
To be exhaustive, you can be called without a "standard" value meaning you are using the native coin (TRX).
A trc10 need to be identified with its "id" (which can be provide through the `contractAddress` field)